config-generators/postgresql-commands.txt

```diff
@@ -104,3 +106,5 @@ update Journal --config "dab-config.PostgreSql.json" --permissions "policy_teste
 update Journal --config "dab-config.PostgreSql.json" --permissions "authorizationHandlerTester:read"
 update ArtOfWar --config "dab-config.PostgreSql.json" --permissions "authenticated:*" --map "DetailAssessmentAndPlanning:始計,WagingWar:作戰,StrategicAttack:謀攻,NoteNum:┬─┬ノ( º _ ºノ)"
 update Sales --config "dab-config.PostgreSql.json" --permissions "authenticated:*"
+update Bookmarks --config "dab-config.PostgreSql.json" --permissions "authenticated:*"
+update MappedBookmarks --config "dab-config.PostgreSql.json" --permissions "authenticated:*" --map "id:bkid,bkname:name"
```

Comment:
With the addition of the CONTRIBUTING.md PR, it seems the checked in configs would be used for testing in the pipeline. So, I think you would need to modify the checked in configs. 

The purpose of config generator scripts has now become config validation script now...

Reply:
Would you prefer I exclude the generator script changes from this PR?

Reply:
ah just saw "Given the checked in config files in the Test project, we may remove these reference files altogether"

Reply:
since this PR does not have removing config files in scope, will save for separate PR. So will update existing config files until removed.